influxql/parser.go

```diff
@@ -1403,6 +1403,13 @@ func (p *Parser) parseCreateContinuousQueryStatement() (*CreateContinuousQuerySt
 	}
 	stmt.Database = ident
 
+	if p.parseTokenMaybe(RESAMPLE) {
```

Comment:
Why `parseTokenMaybe` here? Instead of

```
tok, _, _ := p.scanIgnoreWhitespace()
if tok == RESAMPLE {
    stmt.ResampleEvery, stmt.ResampleFor, err = p.parseResample()
} else {
    p.unscan()
}
```